slam_toolbox/launch/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 PAL Robotics S.L.
```

Comment:
This project is LGPLv2.1 due to the use of openkarto, please use the appropriate license header (or dont include like the others)